app/code/Magento/Sales/Api/Data/OrderItemInterface.php

```diff
@@ -414,6 +414,7 @@ public function getAmountRefunded();
 
     /**
      * Gets the applied rule IDs for the order item.
+     * Rules are comma separated if there are more than one.
```

Comment:
Please include a blank line before or add the "(comma separated)" information in return line, removing this one.